### DISCUSSION TITLE
Call to jira.transitions() always return an empty list

### DISCUSSION THREAD

**Discussion by dsforetellix** [pycontribs/jira#1822] [Q&A] [answered]
Hi,
Whatever the Jira current status, the call to jira.transitions() returns [].

In debug mode, I've checked the URL that the SDK sends: 
`https://<jira-url>/rest/api/2/issue/<issue>/transitions`

- when the SDK calls this URL, the response code is 200, but the content is []
- when I enter that URL in my browser I get all the possible transitions perfectly

Any idea on what's causing this issue ?

Thanks
David

**Comment by dsforetellix** [comment, marked as answer]
I was not using the same user for the SDK, and the IT team did not give this user the proper permissions. Sad that I get an empty list instead of an error ...

**Reply by mdoar** [reply]
Yeah, consistency is not a strong point. Try looking for the number of issue security scheme levels associated with a project. When there are no levels, which is a valid case, you get an error
